servant/src/Servant/API/Modifiers.hs

```diff
@@ -131,7 +132,8 @@ type RequestArgument mods a =
        (If (FoldLenient mods) (Either Text a) a)
        (Maybe (If (FoldLenient mods) (Either Text a) a))
 
-
+type LenientArgument mods a =
```

Comment:
This is not actually used? If so, I think you should just remove it.

Reply:
Fixed.